rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RMNodeStarter.java

```diff
@@ -432,6 +435,63 @@ public static void main(String[] args) {
         }
     }
 
+    private static BlockingQueue<KeyPair> keyPairs = new LinkedBlockingQueue<>();
+
+    private static void launchBackgorundKeyPairMaker(final int numberOfWorkers) {
+        final Thread thread = new Thread(new Runnable() {
```

Comment:
We can probably use a single thread ExecutorService in place of the thread. Also, it would be good to give a name this thread so that we know what it is when troubleshooting

Reply:
Agree about the name, the Thread must be daemon also, I don't think an ExecutorService is necessary.